pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -115,12 +106,27 @@ type ClusterDeploymentSpec struct {
 
 	// Installed is true if the cluster has been installed
 	Installed bool `json:"installed"`
+
+	// Provisioning contains settings used only for initial cluster provisioning.
+	Provisioning Provisioning `json:"provisioning"`
```

Comment:
Should this be optional to accommodate adopting installed clusters?

Reply:
Yes good idea.